ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java

```diff
@@ -13625,7 +13656,8 @@ ASTNode analyzeCreateTable(
       }
       tblProps = validateAndAddDefaultProperties(
           tblProps, isExt, storageFormat, dbDotTab, sortCols, isMaterialization, isTemporary,
-          isTransactional, isManaged);
+          isTransactional, isManaged, new String[] {qualifiedTabName.getDb(), qualifiedTabName.getTable()}, isDefaultTableTypeChanged);
```

Comment:
its not clear where the value for isDefaultTableTypeChanged is coming from in analyzeCreateTable method. The "View file" only shows about 12k lines (a truncated version of this file)

Reply:
This flag is used as one of the determining factors in the validateAndAddDefaultProperties() to decide whether to create an acid table or managed table.